Vagrantfile

```diff
@@ -12,6 +12,8 @@ Vagrant.configure("2") do |config|
         owner: 'root', group:'vboxsf', automount:'true', 
         mount_options: ["dmode=0770", "fmode=0770"]
 
+    config.vm.network "private_network", ip: "192.168.42.100"
```

Comment:
This addition prompts for elevated user approvals that VirtualBox make changes to the machine, I'm not sure we can assume that for NOS3.

Reply:
Agree... we will discuss and I will look into this more.